### PR TITLE
Add support for the CryptoTari stellar token

### PR DESCRIPTION
**Pull request by litlp007**
Block explorer: https://stellarchain.io/
The TARI token is issued by account GD7UVDDJHJYKUXB4SJFIC6VJDQ4YADQCMRN3KLHJFV4H6NIUAEREVCO7 on the stellar network, it's toml file can be accessed at https://cryptotari.io/.well-known/stellar.toml.
This request is a follow-up to request: https://github.com/bisq-network/bisq-assets/pull/19, now with offline address validation.

### CODE REVIEW

File: src/main/java/bisq/asset/tokens/Tari.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+package bisq.asset.tokens;
+
+import bisq.asset.Token;
+import bisq.asset.StellarAddressValidator;
+
+public class Tari extends Token {
+
+    public Tari() {
+        super("CryptoTari", "TARI", new StellarAddressValidator() );
```
Comment (blabno) [inline src/main/java/bisq/asset/tokens/Tari.java]: Use existing regex validator.

File: src/main/java/bisq/asset/tokens/Tari.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ *
+ *
+ * CryptoTari is a stellar token issued by account:
+ * GD7UVDDJHJYKUXB4SJFIC6VJDQ4YADQCMRN3KLHJFV4H6NIUAEREVCO7
+ * block explorer: https://stellarchain.io/
+ */
+
+package bisq.asset.tokens;
+
+import bisq.asset.Token;
```
Comment (blabno) [inline src/main/java/bisq/asset/tokens/Tari.java]: This file does not conform to Bisq code style.

Comment (litlp007) [reply src/main/java/bisq/asset/tokens/Tari.java]: I have reworked to use the Regex validator, indeed it is indeed a better choice.

File: src/main/java/bisq/asset/tokens/Tari.java (hunk 1 above)
Comment (litlp007) [reply src/main/java/bisq/asset/tokens/Tari.java]: I have reworked to use the Regex validator, indeed it is indeed a better choice.

File: src/main/java/bisq/asset/tokens/Tari.java (hunk 2 above)
Comment (litlp007) [reply src/main/java/bisq/asset/tokens/Tari.java]: I have changed the commit for the other comment but can you please explain what is the coding style error here.

Comment (litlp007) [reply src/main/java/bisq/asset/tokens/Tari.java]: @blabno ?

**Pull request merged**